third_party/move/move-compiler/src/expansion/ast.rs

```diff
@@ -421,6 +421,12 @@ pub type LValue = Spanned<LValue_>;
 pub type LValueList_ = Vec<LValue>;
 pub type LValueList = Spanned<LValueList_>;
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct TypedLValue_(pub LValue, pub Option<Type>);
```

Comment:
Perhaps add a brief comment here saying these are used to represent explicitly typed (by the developer) lvalues.

Reply:
Done